libcxx/include/__config

```diff
@@ -1198,6 +1198,8 @@ typedef __char32_t char32_t;
 #    define _LIBCPP_DISABLE_UBSAN_UNSIGNED_INTEGER_CHECK
 #  endif
 
+// TODO(LLVM 22): Remove this macro once LLVM19 support ends. __cpp_explicit_this_parameter has been set in LLVM20
+// https://github.com/llvm/llvm-project/pull/107451
```

Comment:
I don't mind the link, but I don't think it's really required when doing similar patches in the future.

Reply:
Thanks!